### PR TITLE
use grpc client for wait request

### PR DESCRIPTION
**Pull request by kolsa**

### CODE REVIEW

**Comment by majst01** [comment]
Shouldn´t we create a metal-api grpc client package, like in masterdata-api which can be directly used by consumers instead of recreating the client logic there as well ?

**Comment by kolsa** [comment]
Good point. I have postponed this issue, but would be also fine with me to do it right now together with this PR

File: main.go [hunk 1]
```diff
@@ -18,30 +17,46 @@ import (
...
 		log.Error("sshd error", "error", err)
 		os.Exit(1)
 	}
 
-	if len(os.Args) > 1 {
-		log.Error("cmd args are not supported")
-		os.Exit(1)
-	}
+	// Reboot if metal-hammer crashes after 60sec.
+	go kernel.Watchdog()
```
Comment (Gerrit91) [inline main.go:52]: Why did the position of the watchdog start change in the code?

File: cmd/register/register.go [hunk 2]
```diff
@@ -203,7 +203,7 @@ const (
 	defaultIpmiUser = "metal"
 
 	// defaultIpmiUserID the default id of the created ipmi user
-	defaultIpmiUserID = "10"
+	defaultIpmiUserID = "2"
```
Comment (majst01) [inline cmd/register/register.go:206]: I would recommend to remove this once the lenovo test is done and move to go-hal

Comment (kolsa) [reply cmd/register/register.go:206]: Good point. We should also move the channel number and username

**Pull request merged**